scripts/qa_crowbarsetup.sh

```diff
@@ -1414,47 +1456,73 @@ function enable_ssl_for_nova_dashboard()
     proposal_set_value nova_dashboard default "['attributes']['nova_dashboard']['ssl_no_verify']" true
 }
 
-function hacloud_configure_cluster_defaults()
+function hacloud_configure_cluster_members()
 {
-    clustertype=$1
+    local operation=$1
+    local clustername=$2
     shift
+    shift
+    # default is to add nodes
+    local op="+="
```

Comment:
no you are right its not that important here, maybe we should just tag the functions that do ruby with `ruby_proposal_modify_value` or so. that helps us to slowly migrate the whoooole thing to ruby in the end ;)